manifests/l/LAMMPS/LAMMPS-USER-VCSGC/23Jun2022/LAMMPS.LAMMPS-USER-VCSGC.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LAMMPS.LAMMPS-USER-VCSGC
+PackageVersion: 23Jun2022
+PackageLocale: en-US
+Publisher: The LAMMPS and USER-VCSGC Developers
```

```suggestion
Publisher: The LAMMPS and PACE Developers
```
This must match Control Panel; otherwise WinGet won't be able to recognize this application.

---

Ah, now I understand. This is a problem with the NSIS installer package script. I need to correct and recreate the installer.